CHANGELOG.md

```diff
@@ -7,15 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 **Please Note**
+- In order for Cumulus to produce reports of files accessed via your distribution API, you must set up S3 access logging on all of your protected buckets. See [documentation on EMS reporting](https://nasa.github.io/cumulus/docs/ems_reporting) for how to correctly set up S3 access logging for your deployment.
```

Comment:
Did anything change here that we are calling this out again?

Reply:
Nope. Patrick just thought it was worth mentioning, because originally he had wanted to have this turned on by default. But we can't do that since we're not deploying buckets.

Shall I remove it?

Reply:
I just think it's kind of random because we have it further down in the changelog and the docs, so probably because it makes me feel like something has changed reading it.